so/challenges/default.json

```diff
@@ -48,6 +101,21 @@
             "retry-until-success": true,
             "include-in-reporting": false
           }
+        },
+        {
+          "operation": "frequent_items_all_100",
+          "warmup-iterations": 3,
+          "iterations": 10
```

Comment:
Just a drive-by question, are these particularly long-running or intensive queries? You will only have 10 deciles if you only have  10 iterations, FYI, so we normally run >=100 ops in order to get a good sense of the distribution (unless it would cause the benchmark to run too long)

Reply:
yes,  `frequent_items` is quite heavy. On the other hand I don't want to simplify the query or reduce the data. Reducing iterations seems therefore reasonable, however we can discuss whether it is 10 or 20 or 50. What's an acceptable total runtime in your opinion?

Reply:
> What's an acceptable total runtime in your opinion?

I don't think there's a correct answer to this question, it's a matter of preference. How much variance is there in your measured service time? If there's a wide distribution you may want more iterations, and if not then you might be fine with 10.

Reply:
There isn't a wide distribution. For the purpose of e.g. finding performance regressions or to proof/disproof enhancements 10 is sufficient.